src/utils/init.ts

```diff
@@ -212,14 +198,14 @@ function setValues(userVals: FieldValues, rerender = true): void {
 }
 
 function setAuthClient(client: any): void {
```

Comment:
maybe move these functions to a utils file at /auth/utils.ts

(along with /auth/LoginProvider.tsx, /auth/useFormAuth.ts, etc.)